tangelo/linq/circuit.py

```diff
@@ -173,6 +179,10 @@ def check_index_valid(index):
         # Keep track of the total gate count
         self._gate_counts[gate.name] = self._gate_counts.get(gate.name, 0) + 1
 
+        # Keep track of n_qubit gate count
```

Comment:
Usually I'm pretty concise but I think it would be clearer for the reader if the comment was something like

`# Keep track of gate counts separately for 1-qubit gates, 2-qubit gates, and so on`